pyproject.toml

```diff
@@ -84,17 +85,20 @@ include = [
     "./apps/prairielearn/elements/pl-variable-output",
     "./apps/prairielearn/elements/pl-workspace",
     "./apps/prairielearn/elements/pl-xss-safe",
-    "./apps/prairielearn/python/check_data.py",
     "./apps/prairielearn/python/check_data_test.py",
-    "./apps/prairielearn/python/conftest.py",
+    "./apps/prairielearn/python/check_data.py",
     "./apps/prairielearn/python/colors.py",
-    "./apps/prairielearn/python/prairielearn.py",
+    "./apps/prairielearn/python/conftest.py",
     "./apps/prairielearn/python/prairielearn_test.py",
-    "./apps/prairielearn/python/python_helper_sympy.py",
+    "./apps/prairielearn/python/prairielearn.py",
     "./apps/prairielearn/python/python_helper_sympy_test.py",
+    "./apps/prairielearn/python/python_helper_sympy.py",
     "./apps/prairielearn/python/question_phases.py",
-    "./apps/prairielearn/python/traverse.py",
     "./apps/prairielearn/python/traverse_test.py",
+    "./apps/prairielearn/python/traverse.py",
 ]
 extraPaths = ["./apps/prairielearn/python"]
 pythonVersion = "3.9"
+
+[tool.pytest.ini_options]
+addopts = "--ignore=./pl_ag_jobs"
```

Comment:
I think we should remove this. I'm ok with adding something to the gitignore to avoid accidentally committing something, but we should try to have only what is necessary in actual tooling configuration.